docs/example-scenario/infrastructure/multi-tier-app-disaster-recovery.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Highly available (HA) and disaster recovery (DR) configured multi-tier web application
+description: Build a highly available and disaster recovery configured multi-tier web application on Azure using Azure virtual machines, Availability sets, Availability zones, Azure Site Recovery and Azure Traffic Manager
+author: sujayt
+ms.date: 11/16/2018
+---
+
+# Build a highly available and disaster recovery configured multi-tier web application
+
+This example scenario is applicable to any industry that have a need to deploy resilient multi-tier applications with high availability and disaster recovery configured. In this scenario, the application consists of 3 layers.
+
+- Web tier - This is the top most layer with user interface. This translates inputs and outputs into something user understands.
+- Business tier - This layer processed the user interactions and makes logical decisions about the next steps. It acts as a connection between the web tier and the data tier
+- Data tier - This is the layer where the data is stored. Typically, a database or a file storage is used.
```

Comment:
Maybe instead of "a database or a file storage is used", "a database, object storage, or file storage is used."

Reply:
changed